models.py

```diff
@@ -602,12 +602,15 @@ def is_accessible(self):
                 current_user.is_authenticated and
                 self.check_roles())
 
-class UserModelView(RestrictedModelView):
+class AdminUserModelView(RestrictedModelView):
+    can_create = True
     can_delete = False
     can_edit = False
 
-    def can_create(self):
-        return current_user.has_role('admin')
+    def is_accessible(self):
```

Comment:
This is unnecessary, already in parent class

Reply:
Not sure how thats going to work essentially, by placing this to use the FinanceUserModelView(AdminUserModelView), your giving Admin access to the view as well. Would that cause problem if I am admin and try to connect to the create button?

Reply:
looks like its creating 2 "user" when you login to an account with admin role. Not, good.